### PR TITLE
CASMINST-4684 Remove Manual Wipe Steps

### PR DESCRIPTION
**Pull request by rustydb**
# Description

With the fix from MTL-1288, manually wiping is no longer necessary. To rebuild an NCN after following respective procedures for expunging the NCN from its clusters, only `metal.no-wipe=0` must be set and the node PXE booted.

# Checklist Before Merging

- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [x] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I don't have a JIRA.

[1]: https://github.com/Cray-HPE/docs-csm/blob/MTL-1695/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: operations/node_management/Wipe_NCN_Disks.md [hunk 1]
```diff
@@ -6,13 +6,15 @@ This page details how to wipe NCN disks.
...
 
 The following are potential use cases for wiping disks:
 
-* Adding a node that is not bare.
+* Adding a node that is not bare-metal, that came from somewhere else and might have a running OS already
 * Adopting new disks that are not bare.
-* Doing a fresh install.
+
+By default the NCN is wiped by its own deployment code (when `metal.no-wipe=0` is set), this 
+procedure in this page is offered as an alternative, manual method. This page is not required.
```
Comment (mharding-hpe) [inline operations/node_management/Wipe_NCN_Disks.md:17]:
```suggestion
procedure in this page is offered as an alternative, manual method.
```

File: operations/node_management/Wipe_NCN_Disks.md [hunk 2]
```diff
@@ -6,13 +6,15 @@ This page details how to wipe NCN disks.
 
 After following these procedures, an NCN can be rebooted and redeployed.
 
-All types of disk wipe can be run from Linux or from an initramFS/initrd emergency shell.
+> **`NOTE`** All types of disk wipe can be run from Linux or from an initramFS/initrd emergency shell.
```
Comment (mharding-hpe) [inline operations/node_management/Wipe_NCN_Disks.md:9]:
```suggestion
> **NOTE** All types of disk wipe can be run from Linux or from an `initramFS` or `initrd` emergency shell.
```

File: operations/node_management/Add_Remove_Replace_NCNs/Remove_NCN_from_Role.md [hunk 3]
```diff
@@ -324,91 +320,6 @@ Open a new tab and follow [Remove Ceph Node](../../utility_storage/Remove_Ceph_N
 
 Once the storage node role removal is complete, then proceed to [wipe the drives](#wipe-disks-utility-storage-node).
```
Comment (mharding-hpe) [inline operations/node_management/Add_Remove_Replace_NCNs/Remove_NCN_from_Role.md:321]:
```suggestion
Once the storage node role removal is complete, then proceed to [Power Off the Node](#power-off-the-node).
```

**Review by mharding-hpe — APPROVED** [review]

**Review by mharding-hpe — APPROVED** [review]

**Review by heemstra — APPROVED** [review]

**Comment by rustydb** [comment]
> I did some linting. Among other things, this cleared up the stuff the linter was barking about.

I might've overwritten those but I'll factor them back in. Was going to appease the linter after I was done with content changes, I have a few more changes to the upgrade script incoming.

**Comment by rustydb** [comment]
I still need to add a piece to the rebuild script that mitigates the possibly of the node disk-booting during a rebuild (since wiping is removed).

**Review by heemstra — APPROVED** [review]

**Pull request merged**